pkg/adapterManager/registry.go

```diff
@@ -20,20 +20,19 @@ import (
 	"github.com/golang/glog"
 	"istio.io/mixer/pkg/adapter"
 	"istio.io/mixer/pkg/aspect"
-	"istio.io/mixer/pkg/config"
 )
 
 // registry implements pkg/adapter/Registrar.
 // registry is initialized in the constructor and is immutable thereafter.
 // All registered builders must have unique names.
 // It also implements builderFinder that manager uses.
 type registry struct {
-	buildersByName map[string]adapter.Builder
+	builders map[string]map[string]adapter.Builder
```

Comment:
Maybe we could have a type named KindNameBuilderMap or something to document it here and the other places in the code?